### PR TITLE
Add README on how to run the gatling pugling

### PR DESCRIPTION
**Pull request by eleonoracau**
- Added a README.md file with instructions on how to run the Gatling plugin
- Added a jss7.patch needed for the stack to work with the STP setup

### CODE REVIEW

File: gatling-ss7/README.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+```` 
+
+- Install and copy the fat jar to the target host in `dev` (just clone a visited-plmn-lte instance)
+
+````
+mvn install
+owl scp ./gatling-engine/target/snice-gatling-engine-0.0.5-SNAPSHOT.jar wireless-mobilecore-visited-plmn-lte:/tmp
+````
+
+- Ensure the target STP host has the Gatling Peer properly configured in the _stp.xml_. If not, add it with the Private Ip of the Gatling host and restart the STP
```
Comment (eleonoracau) [inline gatling-ss7/README.md:40]: Kuba, I would suggest to add a permanent peer config for dev in STP as we do already for MAP_TEST. wdyt?

Comment (jko881) [reply gatling-ss7/README.md:40]: Good idea, should I do that or you will?

File: gatling-ss7/README.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+  val REMOTE_IP = "172.22.138.96" // Private Ip of the STP host
+  val REMOTE_PORT = 2905
+```` 
+
+- Install and copy the fat jar to the target host in `dev` (just clone a visited-plmn-lte instance)
+
+````
+mvn install
+owl scp ./gatling-engine/target/snice-gatling-engine-0.0.5-SNAPSHOT.jar wireless-mobilecore-visited-plmn-lte:/tmp
```
Comment (jko881) [inline gatling-ss7/README.md:37]: We should probably use the wireless-mobilecore-visited-plmn-2g3g role instead.

File: gatling-ss7/README.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+````
+
+- Ensure the target STP host has the Gatling Peer properly configured in the _stp.xml_. If not, add it with the Private Ip of the Gatling host and restart the STP
+````
+    <Peer
+          name="Gatling-TEST"
+          role="wireless-mobilecore-visited-plmn-lte"
```
Comment (jko881) [inline gatling-ss7/README.md:44]: wireless-mobilecore-visited-plmn-2g3g

File: gatling-ss7/README.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+          enable_rkm="0"
+          ipx_id="test-ipx"
+          metric="100"
+          weight="100"
+    />
+````
+
+- ssh into the machine and make sure the host runs with `java8`
+````
+owl ssh wireless-mobilecore-visited-plmn-lte
```
Comment (jko881) [inline gatling-ss7/README.md:64]: wireless-mobilecore-visited-plmn-2g3g

File: gatling-ss7/README.md [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+          message_priority="0"
+          signalling_link_selection="255"
+          enable_rkm="0"
+          ipx_id="test-ipx"
+          metric="100"
+          weight="100"
+    />
+````
+
+- ssh into the machine and make sure the host runs with `java8`
```
Comment (jko881) [inline gatling-ss7/README.md:62]: Could there be any other applications on the machine that would be affected by this change?

File: gatling-ss7/README.md [hunk 6]
```diff
@@ -0,0 +1,77 @@
+How to create a basic (manual) setup to test STP using Gatling-SIGTRAN plugin
+
+- Clone the Jss7 repository
+
+````
+git clone https://github.com/RestComm/jss7.git
+````
+
+- Download the _jss7.patch_ and apply it to the jss7 repo and install
```
Comment (jko881) [inline gatling-ss7/README.md:9]: Could you shortly describe what this patch is about?

**Pull request merged**